.github/workflows/tests.yml

```diff
@@ -96,3 +114,5 @@ jobs:
       - name: Run specs
         run: |
           bundle exec rspec
+      - name: Run Specs With ActiveRecordWithLock Adapter
+        run: ADAPTER="ActiveRecordWithLock" bundle exec rspec
```

Comment:
Could we make this a different job? That way we can put the `ADAPTER` in its own env, and it's useful to see that the specs passed for one adapter and not for the other, rather than failing the whole job